mcpserver/tools.go

```diff
@@ -215,6 +217,10 @@ DO NOT manually install toolchains inside the environment, instead explicitly ca
 		mcp.WithString("explanation",
 			mcp.Description("One sentence explanation for why this environment is being created."),
 		),
+		mcp.WithString("description",
+			mcp.Description("Short description of the work that is happening in this environment. Keep this description updated using `environment_update`."),
```

Comment:
if we do punt on the less-required-arguments environment_update tool, i think we should probably remove "keep this description updated using environment_update" or else some agents might reconfigure the environment too frequently.